src/components/DayPickerRangeController.jsx

```diff
@@ -747,6 +752,23 @@ export default class DayPickerRangeController extends React.Component {
     });
   }
 
+  getNavigationStatus(visibleMonth) {
+    const {
+      minDate,
+      maxDate,
+      numberOfMonths,
+      enableOutsideDays,
+    } = this.props;
+
+    const isMinDateVisible = isDayVisible(minDate, visibleMonth, numberOfMonths, enableOutsideDays);
```

Comment:
If minDate is falsy, it seems like there’s no point in calling isDayVisible?

Reply:
Yep, it will return false in that case.

Happy to change it to make it more readable. I guess it's a good thing not to rely on the method accounting for that scenario (non-moment date)